themes/default/content/docs/get-started/gcp/_index.md

```diff
@@ -13,4 +13,16 @@ aliases: ["/docs/quickstart/gcp/"]
 
 {{< cloud-intro "Google Cloud" >}}
 
+## Guide Goals
+
+In this guide you will:
+
+- Install Pulumi.
+- Install your preferred language runtime.
```

Comment:
Add "Configure access to your Google Cloud account". (and similar for the others?)